notebooks/week3/01.feature_serving.py

```
@@ -0,0 +1,248 @@
+        ]
+    ),
+)
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ### Call The Endpoint
+
+# COMMAND ----------
+
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff

Undefined name `dbutils`

(F821)